From 45fbfcbb393a0d5ac519d2b4b62a0d38de429fb1 Mon Sep 17 00:00:00 2001
From: Tim Stahlhut <stahta01@gmail.com>
Date: Fri, 3 Jun 2022 13:44:33 -0400
Subject: Remove UTF-8 CodePage from manifests

The UCRT64 environment fails to create the default config files when not
removed.
---
 resources/msw/manifests/x64.manifest                | 1 -
 resources/msw/manifests/x64_dpi_aware_pmv2.manifest | 1 -
 resources/msw/manifests/x86.manifest                | 1 -
 resources/msw/manifests/x86_dpi_aware_pmv2.manifest | 1 -
 4 files changed, 4 deletions(-)

diff --git a/resources/msw/manifests/x64.manifest b/resources/msw/manifests/x64.manifest
index 6b11d6f504..0c65d72773 100644
--- a/resources/msw/manifests/x64.manifest
+++ b/resources/msw/manifests/x64.manifest
@@ -20,7 +20,6 @@
   </dependency>
   <asmv3:application>
     <asmv3:windowsSettings>
-      <activeCodePage xmlns="http://schemas.microsoft.com/SMI/2019/WindowsSettings">UTF-8</activeCodePage>
       <longPathAware xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">true</longPathAware>
     </asmv3:windowsSettings>
   </asmv3:application>
diff --git a/resources/msw/manifests/x64_dpi_aware_pmv2.manifest b/resources/msw/manifests/x64_dpi_aware_pmv2.manifest
index ee246a65b1..36454a0d50 100644
--- a/resources/msw/manifests/x64_dpi_aware_pmv2.manifest
+++ b/resources/msw/manifests/x64_dpi_aware_pmv2.manifest
@@ -14,7 +14,6 @@
   </trustInfo>
   <asmv3:application>
     <asmv3:windowsSettings>
-      <activeCodePage xmlns="http://schemas.microsoft.com/SMI/2019/WindowsSettings">UTF-8</activeCodePage>
       <dpiAwareness xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">PerMonitorV2, system</dpiAwareness>
       <dpiAware xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">true</dpiAware>
       <longPathAware xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">true</longPathAware>
diff --git a/resources/msw/manifests/x86.manifest b/resources/msw/manifests/x86.manifest
index c0de5e0970..b471641d26 100644
--- a/resources/msw/manifests/x86.manifest
+++ b/resources/msw/manifests/x86.manifest
@@ -20,7 +20,6 @@
   </dependency>
   <asmv3:application>
     <asmv3:windowsSettings>
-      <activeCodePage xmlns="http://schemas.microsoft.com/SMI/2019/WindowsSettings">UTF-8</activeCodePage>
       <longPathAware xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">true</longPathAware>
     </asmv3:windowsSettings>
   </asmv3:application>
diff --git a/resources/msw/manifests/x86_dpi_aware_pmv2.manifest b/resources/msw/manifests/x86_dpi_aware_pmv2.manifest
index e6c712ed87..93a0b560ce 100644
--- a/resources/msw/manifests/x86_dpi_aware_pmv2.manifest
+++ b/resources/msw/manifests/x86_dpi_aware_pmv2.manifest
@@ -14,7 +14,6 @@
   </trustInfo>
   <asmv3:application>
     <asmv3:windowsSettings>
-      <activeCodePage xmlns="http://schemas.microsoft.com/SMI/2019/WindowsSettings">UTF-8</activeCodePage>
       <dpiAwareness xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">PerMonitorV2, system</dpiAwareness>
       <dpiAware xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">true</dpiAware>
       <longPathAware xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">true</longPathAware>
-- 
